templates/edit.html

```diff
@@ -75,12 +75,12 @@ <h4>
                 </a>
                 </br>
                 <small>
-                    <%= date_added %> -
+                    <%= published_on %> -
                     <a href="<%= source_url %>" target="_blank">
                         <%= publication %>, <%= publication_country %>
                     </a> -
                     <% if (accessed_on !== '' && archive_url){ %>
```

Comment:
Since we're no longer using `accessed_on`, I think this check is out of date. We can probably get away with removing the conditional logic altogether and leaving only this line:

```html
{% trans "Access Point" %} <a href="<%= archive_url %>"><%= access_point %></a>
```